g/grails/grails_ubi8.5.sh

```diff
@@ -0,0 +1,56 @@
+# -----------------------------------------------------------------------------
+#
+# Package		: grails
+# Version		: v5.0.2
+# Source repo	: https://github.com/grails/grails-core.git
+# Tested on	: ubi 8.5
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Sapna Shukla <Sapna.Shukla@ibm.com>
```

Comment:
Please add `Language` field in Script header.

Reply:
Added.